cmd/commands/service/command.go

```diff
@@ -114,16 +111,9 @@ type serviceCommand struct {
 func (sc *serviceCommand) Run(ctx *cli.Context) (err error) {
 	serviceTypes := make([]string, 0)
 
-	// TODO: migration. remove later https://github.com/mysteriumnetwork/payments/issues/234
-	activeServices := config.Current.GetString(config.FlagActiveServices.Name)
-	if len(activeServices) > 0 {
-		serviceTypes = strings.Split(activeServices, ",")
-	} else { // start migration
-		if len(config.Current.GetString(config.FlagWireguardAccessPolicies.Name)) > 0 || len(config.Current.GetString(config.FlagAccessPolicyList.Name)) > 0 {
-			serviceTypes = []string{scraping.ServiceType, datatransfer.ServiceType}
-		} else {
-			serviceTypes = []string{wireguard.ServiceType, scraping.ServiceType, datatransfer.ServiceType}
-		}
+	fromConfig := config.Current.GetString(config.FlagActiveServices.Name)
```

Comment:
i suggest to keep the name `activeServices`. `fromConfig` could be a bit confusing, cause value will be fetched from different sources, not only from config file